### DISCUSSION TITLE
2FA SS7 farce

### DISCUSSION THREAD

**Discussion by 86ed** [community/community#129721] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

<img width="765" alt="no I will not share a mobile number you need to accept voip" src="https://github.com/community/community/assets/138184084/f215ed05-78a7-4a33-a87a-88aeb0f5e2f1">

When will github accept voip as my 2FA conduit?  I will not share a mobile number.

**Comment by ebndev** [comment]
Hi @86ed, thanks for being a part of the GitHub Community!

If you do not want to share your mobile number, you can use a TOTP app to configure 2FA as well. Please see our [documentation](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-a-totp-app) for more information.
